spaceship.zsh

```diff
@@ -370,10 +370,9 @@ spaceship_golang_version() {
 
   command -v go > /dev/null 2>&1 || return
 
-  # How may .go files are in current directory?
-  local gofiles=$(ls -1 *.go 2>/dev/null | wc -l)
+  local gofiles=(*.go(N))
   # If there are Go-specific files in current directory
-  if [[ $gofiles != 0 || -d Godeps || -f glide.yaml ]]; then
+  if [[ -n $gofiles || -d Godeps || -f glide.yaml ]]; then
```

Comment:
I tried it without using the variable but could not make it works. It gives true even when there are no Go-files in the directory.

In [Zsh's doc](http://zsh.sourceforge.net/Doc/Release/Conditional-Expressions.html), it says
> Filename generation is not performed on any form of argument to conditions...

So if I run the following,
```
[[ -n *.go(N) ]] && echo "true"
```
it will always give `true` regardless of existence of Go-files in the directory.